src/stan-users-guide/odes.Rmd

```diff
@@ -209,6 +209,7 @@ These ten noisy observations of the state can be used to estimate the friction
 parameter, $\theta$, the initial conditions, $y(t_0, \theta)$, and the scale of
 the noise in the problem. The full Stan model is:
 
+\newpage
```

Comment:
Seems hard to manage these manually. Is this something we do before huge code blocks?

Reply:
not generally.
this isn't a problem in the html versions - just the pdf where images and captions float away from where they are in the text and will then interrupt long code blocks - a better solution is to find a way to not let things float into code blocks in the pdfs.
leave in or remove - your call.

Reply:
If it works, I'm happy with it. The pdf looked fine to me.